### PR TITLE
SEO-538 Revert a tweak to return 410 HTTP code for deleted articles

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [SEOTweaksHooksHelper.class.php#L38](https://github.com/Wikia/app/blob/1ea0975c1b51138e38b69744915602ad6bb63bfc/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L38): Remove the unused function parameter "$html". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [SEOTweaksHooksHelper.class.php#L72](https://github.com/Wikia/app/blob/1ea0975c1b51138e38b69744915602ad6bb63bfc/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L72): Remove the unused function parameters "$parser, $params, $time, $descQuery, $options". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [SEOTweaksHooksHelper.class.php#L90](https://github.com/Wikia/app/blob/1ea0975c1b51138e38b69744915602ad6bb63bfc/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L90): Remove the unused function parameter "$pcache". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. INFO [SEOTweaksHooksHelper.class.php#L76](https://github.com/Wikia/app/blob/1ea0975c1b51138e38b69744915602ad6bb63bfc/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L76): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 48 issues
* MAJOR 9 major
* MINOR 27 minor
* INFO 12 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [SEOTweaksHooksHelper.class.php#L38](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L38): Remove the unused function parameter "$html". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [SEOTweaksHooksHelper.class.php#L72](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L72): Remove the unused function parameters "$parser, $params, $time, $descQuery, $options". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [SEOTweaksHooksHelper.class.php#L90](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L90): Remove the unused function parameter "$pcache". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
1. MAJOR [SEOTweaksTest.php#L101](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L101): Remove this unused "$mockTitle" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SEOTweaksTest.php#L106](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L106): Remove this unused "$mockFile" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SEOTweaksTest.php#L170](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L170): Remove this unused "$mockWrapper" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SEOTweaksTest.php#L270](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L270): Remove this unused "$mockHandler" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. MAJOR [SEOTweaksTest.php#L576](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L576): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. MAJOR [SEOTweaksTest.php#L609](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L609): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. MINOR [SEOTweaksTest.php#L20](https://github.com/Wikia/app/blob/7d8575cbaccc5bb9659819f3ccdfa41f0cd2f582/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L20): Define a constant instead of duplicating this literal "OutputPage" 6 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 48 issues
- MAJOR 9 major
- MINOR 27 minor
- INFO 12 info

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [SEOTweaksHooksHelper.class.php#L38](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L38): Remove the unused function parameter "$html". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
2. MAJOR [SEOTweaksHooksHelper.class.php#L72](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L72): Remove the unused function parameters "$parser, $params, $time, $descQuery, $options". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
3. MAJOR [SEOTweaksHooksHelper.class.php#L90](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php#L90): Remove the unused function parameter "$pcache". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
4. MAJOR [SEOTweaksTest.php#L101](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L101): Remove this unused "$mockTitle" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
5. MAJOR [SEOTweaksTest.php#L106](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L106): Remove this unused "$mockFile" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
6. MAJOR [SEOTweaksTest.php#L170](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L170): Remove this unused "$mockWrapper" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
7. MAJOR [SEOTweaksTest.php#L270](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L270): Remove this unused "$mockHandler" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
8. MAJOR [SEOTweaksTest.php#L576](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L576): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
9. MAJOR [SEOTweaksTest.php#L609](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L609): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
10. MINOR [SEOTweaksTest.php#L20](https://github.com/Wikia/app/blob/0b40368622c7fc89365d14b6e204259a6566f5e3/extensions/wikia/SEOTweaks/tests/SEOTweaksTest.php#L20): Define a constant instead of duplicating this literal "OutputPage" 6 times. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)

File: extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php [hunk 1]
```diff
@@ -29,41 +29,6 @@ static function onBeforePageDisplay( $out ) {
...
-				case NS_FILE:
-					// skip existing file with deleted description
-					$file = wfFindFile( $title );
-					if ( $file && $file->exists() ) {
-						$setDeletedStatusCode = false;
-					}
-					break;
-			}
-			if ( $setDeletedStatusCode ) {
-				$output->setStatusCode( SEOTweaksHooksHelper::DELETED_PAGES_STATUS_CODE );
```
Comment (saipetch) [inline extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php:60]: consider removing DELETED_PAGES_STATUS_CODE ?

**Comment by saipetch** [comment]
LGTM

**Pull request merged**